packages/core/template/default/index.md

```diff
@@ -1,132 +1,75 @@
 <frontmatter>
-  title: Landing Page
+  title: Home Page
   layout: default.md
-  pageNav: 2
+  pageNav: 4
   pageNavTitle: "Chapters of This Page"
 </frontmatter>
 
 <br>
 
-<div class="bg-primary text-white py-5 mb-4">
+<div class="bg-primary text-white px-2 py-5 mb-4">
   <div class="container">
-    <h1 class="display-4 no-index">Landing Page Title</h1>
-    <p class="lead">A tagline can go here</p>
+    <h1 class="display-5 no-index">Great!<br>You've just initialized a MarkBind site.</h1>
+    <p class="lead">Let's get started...</p>
   </div>
 </div>
 
-# Heading 1
-Some text some text some text some text some text some text some text. **Some text some text some text some text some text ==some text== some text**. Some text some text some text some text some text some text some text some text some text some text some text some text some text some text. Some text some text some text some text some text some text. Some text some text some text some text some text some text some text.
+---
 
-**A block quote:**
+## What did I just do?
```

Comment:
```suggestion
## What did you just do?
```

Reply:
Fixed in [945ea73](https://github.com/MarkBind/markbind/pull/2225/commits/945ea738078da9000f4299903da634c4db9dee5e)!